smsparkbuild/py39/pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [tool.black]
 line-length = 120
-target-version = ['py39']
+target-version = ['py37']
```

Comment:
why are we changing target version to37 for python 3.9?

Reply:
changed by mistake. Changed back.

Reply:
Did not see it get changed, I assume there will be another commit fixing it